libraries/ts-command-line/src/CommandLineHelper.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { CommandLineConstants } from './Constants';
+
+/**
+ * Helpers for working with the ts-command-line API.
+ *
+ * @public
+ */
+export class CommandLineHelper {
+  /**
+   * Returns true if the current command line action is tab-complete.
```

Comment:
This should receive `process.argv` as a parameter, rather than assuming the parser's input is always the global variable.